tracing-example/pom.xml

```diff
@@ -48,6 +46,39 @@
         <debugJvmArgs/>
     </properties>
 
+    <profiles>
+        <profile>
+            <id>wildfly27</id>
+            <properties>
+                <version.org.wildfly>27.0.1.Final</version.org.wildfly>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.jboss.resteasy</groupId>
+                    <artifactId>resteasy-tracing-api</artifactId>
+                    <version>${version.org.jboss.resteasy.extensions}</version>
```

Comment:
This needs a `<scope>runtime</scope>` I think.

Reply:
yes sir!